woocommerce/woocommerce
# [Settings] Fix Sidebar "Back" navigation to return to WooCommerce Home

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/53159

This PR updates the behavior of the "back" chevron in the sidebar navigation to return merchants to the WooCommerce Home screen instead of the main WordPress admin dashboard. 


Currently, this is set by Gutenberg in the EditSite Store, so we have to set the `__experimentalDashboardLink` in the EditSite Store. We should consider updating the upstream to allow passing a prop instead of setting the `__experimentalDashboardLink` in the store.

https://github.com/WordPress/gutenberg/blob/98b3ec03395836887ba3327cf9a16b279296b6c4/packages/edit-site/src/components/sidebar-navigation-screen/index.js#L50




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a fresh site
2. Add/Enable Gutenberg version 19.8.0 or earlier.
3. Turn on the `settings` feature flag
4. Navigate to WooCommerce > Settings screen.
5. Click the back chevron in the sidebar navigation.
6. Verify that the navigation directs you to the WooCommerce Home screen instead of the WordPress admin dashboard.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
